### PR TITLE
[Hub Generated] Review request for Microsoft.HealthcareApis to add version preview/2018-08-20-preview

### PR DESCRIPTION
**Pull request by YazanMSFT**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/YazanMSFT/azure-rest-api-specs/dev-healthcareapis-Microsoft.HealthcareApis-2018-08-20-preview...master).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/preview/2018-08-20-preview/healthcare-apis.json [hunk 1]
```diff
@@ -392,52 +392,6 @@
...
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HealthcareApis/moveResources": {
-      "post": {
-        "tags": [
-          "Proxy"
-        ],
-        "description": "Moves resources to another subscription and/or resource group.",
-        "operationId": "Services_MoveResources",
-        "x-ms-examples": {
-          "ServiceList": {
-            "$ref": "./examples/MoveResourcesPost.json"
```
Comment (nschonni) [inline specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/preview/2018-08-20-preview/healthcare-apis.json]: Think the example file needs to be deleted too

**Comment by YazanMSFT** [comment]
@ravbhatnagar Could you review this please?

**Comment by ravbhatnagar** [comment]
@YazanMSFT - new writable properties have been added to existing api-version. What is the service behavior if these properties are missing in the request body (and say for example they were set previously using portal)

**Comment by YazanMSFT** [comment]
@ravbhatnagar the service is fully backwards compatible in that case, the request will be accepted and their service will continue to behave as it did before.

**Comment by ravbhatnagar** [comment]
Confirmed with @YazanMSFT, in the current modeling, customer can unintentionally wipe out the value set for these new properties. This is not ideal. But seems like this change is already rolled out on the service side. it has been brought to the attention of the team that design first approach must be followed. 
Signing off on this. It will break private preview customers, but service team will communicate it to the current customers.

**Review by amarzavery — APPROVED** [review]

**Pull request merged**